Week2/homework/kalb3/maartjes-work.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+/* cSpell:disable */
+
+const monday = [
+  {
+    name: 'Write a summary HTML/CSS',
+    duration: 180,
+  },
+  {
+    name: 'Some web development',
+    duration: 120,
+  },
+  {
+    name: 'Fix homework for class10',
+    duration: 20,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+];
+
+const tuesday = [
+  {
+    name: 'Keep writing summary',
+    duration: 240,
+  },
+  {
+    name: 'Some more web development',
+    duration: 180,
+  },
+  {
+    name: 'Staring out the window',
+    duration: 10,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+  {
+    name: 'Look at application assignments new students',
+    duration: 40,
+  },
+];
+
+const maartjesTasks = monday.concat(tuesday);
+const maartjesHourlyRate = 20;
+
+function computeEarnings(tasks, hourlyRate) {
+  tasks = tasks
+    .map(durations => durations.duration / 60)
+    .filter(durations => durations >= 2)
+    .map(durations => durations * hourlyRate)
+    .reduce((x, y) => x + y);
```

Comment:
While using `x` and `y` works fine it easier for other developers to know what is going on if you use more descriptive names, `sum`, `taskDuration` for example.